index.html

```diff
@@ -0,0 +1,320 @@
+        resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most Java
+        expression
+        syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed from LiveScript
+        to
+        JavaScript.</p> <br>
+    <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system
+        based
+        on a small number of data types representing numeric, Boolean, and string values. JavaScript has a prototype-based
+        object model instead of the more common class-based object model. The prototype-based model provides dynamic
+        inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without
+        any
+        special declarative requirements. Functions can be properties of objects, executing as loosely typed methods.</p>
+    <br>
+    <p>JavaScript is a very free-form language compared to Java. You do not have to declare all variables, classes, and
+        methods. You do not have to be concerned with whether methods are public, private, or protected, and you do not have
+        to
+        implement interfaces. Variables, parameters, and function return types are not explicitly typed.</p></section>
+       
+         <section id="Hello_world">
+            <h1 id="3">Hello world</h1>
+            <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:</p>
+            <br>
+            <code>
+                        function greetMe(yourName) { alert("Hello " + yourName); }
+                        greetMe("World");
+                    </code>
+            <p>Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!</p>
+         </section>
+         <section id="Variables">
+            <h1 id="4">Variables</h1>
+            <p>You use variables as symbolic names for values in your application. The names of variables, called identifiers,
+                conform
+                to certain rules.</p><br>
+            <p>A JavaScript identifier must start with a letter, underscore (_), or dollar sign ($); subsequent characters can also
+                be
+                digits (0-9). Because JavaScript is case sensitive, letters include the characters "A" through "Z" (uppercase) and
+                the
+                characters "a" through "z" (lowercase).</p><br>
+            <p>You can use ISO 8859-1 or Unicode letters such as å and ü in identifiers. You can also use the Unicode escape
+                sequences
+                as characters in identifiers. Some examples of legal names are Number_hits, temp99, and _name.</p><br>
+         </section>
+         <section id="Declaring_variables">
+            <h1 id="5">Declaring variables</h1><br>
```

Comment:
Please make sure to only have on h1 per page as it makes it easier for the search engines to determine the main header of your page. For any other header on your page, you can h2-h6